lib/config/presets/npm/index.ts

```diff
@@ -19,9 +19,8 @@ export async function getPreset({
 }: PresetConfig): Promise<Preset> {
   let dep;
   try {
-    const { headers, packageUrl } = resolvePackage(packageName);
-    const body = (await http.getJson<NpmResponse>(packageUrl, { headers }))
```

Comment:
It looks like we're not sending the headers anymore. We need these to authentic in Artifactory.

Reply:
They are send on lower http layer. If you have an issues, open a discussion with all details how you are using renovate.

You can reference this PR if you think this broke your workflow.